src/ol/interaction/selectinteraction.js

```diff
@@ -140,6 +156,7 @@ ol.interaction.Select.handleEvent = function(mapBrowserEvent) {
   var features = this.featureOverlay_.getFeatures();
   var /** @type {Array.<ol.Feature>} */ deselected = [];
   var /** @type {Array.<ol.Feature>} */ selected = [];
+  var filter = this.filter_;
```

Comment:
For better readability I'd remove the use of a local `filter` variable in this case.